FilmFestivalLoader/Shared/planner_interface.py

```diff
@@ -376,11 +386,15 @@ def new_film_id(self, key):
         return film_id
 
     def get_film_by_key(self, title, url):
-        film_id = self.film_id_by_key[self._filmkey(title, url)]
-        films = [film for film in self.films if film.filmid == film_id]
-        if len(films) > 0:
-            return films[0]
-        return None
+        try:
+            film_id = self.film_id_by_key[self.film_key(title, url)]
+        except KeyError:
+            raise KeyError(f'title: {title}, url: {url}')
+        else:
+            films = [film for film in self.films if film.filmid == film_id]
+            if len(films) == 0:
+                raise ValueError(f'title: {title}, url: {url}')
```

Comment:
Please consider to give some more information in the raised exceptions, e.g.
- what the actual key is (`title` or `url`)
- why the exception was raised:
  - KeyError: "Key (`key`) not found in film dictionary"
  - ValueError: "Key (`key`) found, but no film found in films list with indicated film ID (`film_id`).

Reply:
Done.